nix/pkgs/haskell/extra.nix

```diff
@@ -57,6 +57,13 @@ let
     inherit compiler-nix-name index-state checkMaterialization;
     plan-sha256 = lib.removeSuffix "\n" (builtins.readFile ./cabal-install.sha);
   };
+  cardanoRepoToolProject = haskell-nix.cabalProject' {
```

Comment:
please do pin the plan-sha256 like we do for the others - this speeds up evaluation quite a bit

Reply:
And if you do that, add it to the script below that updates the sha files!

Reply:
Done